src/script/auth/module/action/ClientAction.ts

```diff
@@ -74,7 +74,6 @@ export class ClientAction {
             ),
           };
 
-      await dispatch(clientAction.doGetAllClients());
```

Comment:
This was way overkill and was called too many times while the data was not consumed

Reply:
#SingleReponsibilityFailure 🙃